docker-compose.yml

```diff
@@ -49,7 +49,7 @@ services:
 
   identity-resolver-service:
     # Replace with local or deployed image
-    image: ghcr.io/pyx-industries/pyx-identity-resolver:latest@sha256:71a02b8b0a2b9c7cc18d33a54060e9af4d543e34bdd6a47c3b3cf4dcadb1711e
+    image: ghcr.io/pyx-industries/pyx-identity-resolver:sha-fa017fb-local@sha256:cd514863bdaf65ab1b47b8e71669a0245b041c861285a83fa7cf0a65789d4e58
```

Comment:
This image is tagged for "local" use?

Reply:
Yep, I don't understand the question.

Reply:
For completeness' sake, Hoa was asking why we are using the local IDR Docker image within the reference implementation.

The reason for this is that the IDR package workflow (the one that builds and publishes the IDR images) produces two images, one for (local, ec2, etc..) use and another for serverless application (AWS lambda).

I agree that the naming convention could be better and will be something we can look at in the future.